routing/handler.go

```diff
@@ -45,6 +45,9 @@ func NewRouter(authService *services.AuthService) *mux.Router {
 	router.HandleFunc("/signup", SignUp(AuthService))
 	router.HandleFunc("/signin", SignIn(AuthService))
 	router.HandleFunc("/signout", SignOut(AuthService))
+	router.HandleFunc("/scooters", getAllScooters)
```

Comment:
I think, you don't need this handlers here (you already have them installed in routing/scooter.go)